.azure/pipelines/blazor-daily-tests.yml

```diff
@@ -0,0 +1,74 @@
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=vsts&tabs=yaml#scheduled
+
+# Daily Tests for Blazor
+# These use Sauce Labs resources, hence they run daily rather than per-commit.
+
+# The only Daily Tests we have run in Sauce Labs and only need to run on one machine (because they just trigger SauceLabs)
+# Hence we use the 'default-build.yml' template because it represents a single phase
+variables:
+  SAUCE_CONNECT_DOWNLOAD_ON_INSTALL: true
+  E2ETESTS_SauceTest: true
+  E2ETESTS_Sauce__TestName: 'Blazor Daily Tests'
+  E2ETESTS_Sauce__TunnelIdentifier: 'blazor-e2e-sc-proxy-tunnel'
+  E2ETESTS_Sauce__HostName: 'sauce.local'
+jobs:
+- template: jobs/default-build.yml
+  parameters:
+    buildDirectory: src/Components
+    isTestingJob: true
+    agentOs: Windows
+    jobName: BlazorDailyTests_Android_Chrome
+    jobDisplayName: "Blazor Daily Tests"
+    afterBuild:
+    # macOS/Chrome
+    - script: 'dotnet test --filter "StandaloneAppTest"'
+      workingDirectory: 'src/Components/test/E2ETest'
+      displayName: 'Run Blazor tests - macOS/Chrome'
+      condition: succeededOrFailed()
+      env:
+        # Secrets need to be explicitly mapped to env variables.
+        E2ETESTS_Sauce__Username: '$(asplab-sauce-labs-username)'
+        E2ETESTS_Sauce__AccessKey: '$(asplab-sauce-labs-access-key)'
+        # Set platform/browser configuration.
+        E2ETESTS_Sauce__PlatformName: 'macOS'
+        E2ETESTS_Sauce__PlatformVersion: '10.14'
+        E2ETESTS_Sauce__BrowserName: 'Chrome'
+        E2ETESTS_Sauce__BrowserVersion: 'latest'
+    # Android/Chrome
+    - script: 'dotnet test --filter "StandaloneAppTest"'
+      workingDirectory: 'src/Components/test/E2ETest'
+      displayName: 'Run Blazor tests - Android/Chrome'
+      condition: succeededOrFailed()
+      env:
+        # Secrets need to be explicitly mapped to env variables.
+        E2ETESTS_Sauce__Username: '$(asplab-sauce-labs-username)'
+        E2ETESTS_Sauce__AccessKey: '$(asplab-sauce-labs-access-key)'
+        # Set platform/browser configuration.
+        E2ETESTS_Sauce__PlatformName: 'Android'
+        E2ETESTS_Sauce__PlatformVersion: '10.0'
+        E2ETESTS_Sauce__BrowserName: 'Chrome'
+        E2ETESTS_Sauce__DeviceName: 'Android GoogleAPI Emulator'
+        E2ETESTS_Sauce__DeviceOrientation: 'portrait'
+        E2ETESTS_Sauce__AppiumVersion: '1.9.1'
+    # iOS/Safari
+    - script: 'dotnet test --filter "StandaloneAppTest"'
+      workingDirectory: 'src/Components/test/E2ETest'
+      displayName: 'Run Blazor tests - iOS/Safari'
+      condition: succeededOrFailed()
+      env:
+        # Secrets need to be explicitly mapped to env variables.
+        E2ETESTS_Sauce__Username: '$(asplab-sauce-labs-username)'
+        E2ETESTS_Sauce__AccessKey: '$(asplab-sauce-labs-access-key)'
+        # Set platform/browser configuration.
+        E2ETESTS_Sauce__PlatformName: 'iOS'
+        E2ETESTS_Sauce__PlatformVersion: '13.0'
```

Comment:
Could this be set to `latest`? Same with Android. These are likely to get stale.

Reply:
SauceLabs supports `latest` only for specific things. Sometimes updating platform version also required updating appium version. I think that is why they don't allow that. We'll need to manually keep this updated.